docs/README.md

```diff
@@ -3,101 +3,102 @@ sidebar_position: 1
 without giving up JSON's uncanny ability to represent eclectic data.
 
 ## Getting Started
 
-Trying out Zed is easy: just [install](install.md) the command-line tool
-[`zq`](commands/zq.md) and run through the [zq tutorial](tutorials/zq.md).
-
-`zq` is a lot like [`jq`](https://stedolan.github.io/jq/)
-but is built from the ground up as a search and analytics engine based
-on the [Zed data model](formats/zed.md).  Since Zed data is a
-proper superset of JSON, `zq` also works natively with JSON.
+Trying out SuperDB is easy: just [install](install.md) the command-line tool
+[`super`](commands/zq.md) and run through the [tutorial](tutorials/zq.md).
 
-While `zq` and the Zed data formats are production quality, the Zed project's
-[Zed data lake](commands/zed.md) is a bit [earlier in development](commands/zed.md#status).
+Compared to putting JSON data in a relational column, the
+[super-structured data model](formats/zed.md) makes it really easy to
+mash up JSON with your relational tables.  The `super` command is a little
+like [DuckDB](https://duckdb.org/) and a little like
+[`jq`](https://stedolan.github.io/jq/) but super-structured data ties the
+two patterns together with strong typing of dynamic values.
 
-For a non-technical user, Zed is as easy to use as web search
-while for a technical user, Zed exposes its technical underpinnings
+For a non-technical user, SuperDB is as easy to use as web search
+while for a technical user, SuperDB exposes its technical underpinnings
 in a gradual slope, providing as much detail as desired,
 packaged up in the easy-to-understand
-[ZSON data format](formats/zson.md) and
-[Zed language](language/README.md).
+[Super JSON data format](formats/zson.md) and
+[SuperPipe language](language/README.md).
+
+While `super` and its accompanying data formats are production quality, the project's
+[SuperDB data lake](commands/zed.md) is a bit [earlier in development](commands/zed.md#status).
 
 ## Terminology
 
-"Zed" is an umbrella term that describes
+"Super" is an umbrella term that describes
 a number of different elements of the system:
-* The [Zed data model](formats/zed.md) is the abstract definition of the data types and semantics
-that underlie the Zed formats.
-* The [Zed formats](formats/README.md) are a family of
-[sequential (ZNG)](formats/zng.md), [columnar (VNG)](formats/vng.md),
-and [human-readable (ZSON)](formats/zson.md) formats that all adhere to the
-same abstract Zed data model.
-* A [Zed lake](commands/zed.md) is a collection of Zed data stored
-across one or more [data pools](commands/zed.md#data-pools) with ACID commit semantics and
-accessed via a [Git](https://git-scm.com/)-like API.
-* The [Zed language](language/README.md) is the system's pipeline language for performing
+* The [super data model](formats/zed.md) is the abstract definition of the data types and semantics
+that underlie the super-structured data formats.
+* The [super data formats](formats/README.md) are a family of
```

Comment:
In a first iteration I had these as "super-structured data model" and "super-structured data formats", but I recognized it was a lot of words and could get unwieldy considering how often we mention the data model in the docs. @mccanne was in favor of dropping the "-structured" and keeping it lowercase "s". We agreed that saying "SuperDB" in this spot would feel odd, and "Super" with a capital "S" is not something we're attempting anywhere else, so "super" basically wins by default at that point.